packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -289,6 +285,7 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> {
         isOpen: true,
         isAnimatingOpen: false
       });
+      this._updateFooterPosition();
```

Comment:
Should `this._updateFooterPosition();` happen after the new state comes in? Or can it happen at the same time. Right now the way you have it react will merge the previous setState 
``` 
       this.setState({
          isOpen: true,
          isAnimatingOpen: false
        });
```
With the one in `this._updateFooterPosition()` which will cause to act as though you did
```
       this.setState({
          isOpen: true,
          isAnimatingOpen: false,
          isFooterSticky: (some value)
        });
```

If you want `this._updateFooterPosition();` to be called after the rerender you should do 
```       
this.setState({
          isOpen: true,
          isAnimatingOpen: false
        }, ()=>this._updateFooterPosition(););
```